test/Core.Test/Services/LicensingServiceTests.cs

```diff
@@ -19,14 +30,16 @@ public class LicensingServiceTests
         private readonly IWebHostEnvironment _hostingEnvironment;
         private readonly ILogger<LicensingService> _logger;
 
+        private static readonly string _organizationDirectory = $"{Environment.CurrentDirectory}/organization";
```

Comment:
For unit testing, should always use a Temp directory for assets directed by the test framework vs. `Environment.CurrentDirectory`, this gives the test framework and any CI runners in the build pipeline to cleanup after test runs if necessary, ensures proper access rights to disk are maintained and that can be overridden by test framework configuration directives, command-line arguments, etc.